app/assets/javascripts/movie.coffee

```diff
@@ -0,0 +1,22 @@
+jQuery ->
+  $(document).on "turbolinks:load", ->
+    chosen_initializer();
+    star_show();
+    star_input();
+
+chosen_initializer = () ->
+  $('.movie-actor-ids').chosen();
+
+@star_show = ()->
```

Comment:
spacing.

Reply:
Its spacing is correct sir. Kindly specify if I am missing something.

Reply:
() -> not ()->